xc/xc7/archs/zynq7/tiles/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ set(IOPAD_M_SITES IOB33M IDELAYE2 ILOGICE3 OLOGICE3)
 
 project_ray_equiv_tile(
   ARCH zynq7
-  TILES RIOPAD_M RIOPAD_S RIOPAD_SING
+  TILES RIOPAD_M RIOPAD_S RIOPAD_SING LIOPAD_M LIOPAD_S LIOPAD_SING
```

Comment:
I think that there was another mistake during my rebase of the `zynq-board` branch.

LIOPADs are not present in the zynq010 device